client/transactions/list/index.js

```diff
@@ -375,6 +400,17 @@ export const TransactionsList = ( props ) => {
 						}
 						autocompleter={ autocompleter }
 					/>,
+					<Button
+						key="download"
+						className="transactions-list__download-button"
+						disabled={ isLoading }
+						onClick={ onDownload }
+					>
+						<Gridicon icon={ 'cloud-download' } />
+						<span>
+							{ __( 'Download', 'woocommerce-payments' ) }
+						</span>
+					</Button>,
```

Comment:
Should we stick to [WC Admin's `classNames`](https://github.com/woocommerce/woocommerce-admin/blob/21d12e711496aaaef3a85236ee92e2ae344f2891/client/analytics/components/report-table/index.js#L425-L436) here to be more consistent with other report pages? We'd also get some additional styling, such as the label being hidden on narrow viewports.

We'd also have to pull the styles for `.woocommerce-table__download-button` from the old [style.css](https://github.com/woocommerce/woocommerce-admin/blob/ec36a00b77b5c0e313985a5a64c88aaec9eb154e/client/analytics/components/report-table/style.scss) to have the button stay on the top row:

![image](https://user-images.githubusercontent.com/7714042/113563498-2de20e80-95de-11eb-9cde-88222599dbbf.png)

I appreciate that we might be shooting ourselves on the foot by copying these styles, so I'd like to get your thoughts on whether we should actually do this.

Reply:
Mi initial thought was to copy the styles from wc-admin, because we have some more copied until a possible migration happens. But then I realize that we don't need all of them to make this simple implementation works.

I miss the stay on top row behavior, thanks for pointing that out.

Maybe the best option now is to copy these styles until we decide if we are going to migrate to wc-admin report component. Because we already are doing that, makes sense. But it's best to point that out in a comment.

What do you think?

Reply:
>Maybe the best option now is to copy these styles until we decide if we are going to migrate to wc-admin report component. Because we already are doing that, makes sense. But it's best to point that out in a comment.

💯 Agreed. We could even copy them under the [already existing comment](https://github.com/Automattic/woocommerce-payments/blob/develop/client/transactions/list/style.scss#L20-L24), by including the [styles](https://github.com/woocommerce/woocommerce-admin/blob/ec36a00b77b5c0e313985a5a64c88aaec9eb154e/client/analytics/components/report-table/style.scss) for `.woocommerce-table__download-button` in the appropriate places. If copying the stay-on-top behavior it'd be nice to rename the button's class and add the label's class, so the text collapses when the table is too narrow.